ui/raidboss/data/05-shb/raid/e10s.js

```diff
@@ -226,9 +223,84 @@ export default {
           cn: '影子放到外圈',
         },
       },
+      run: (data) => data.clones = true,
+    },
+    {
+      // This checks your shadow's job against your job, since your shadow has
+      // the same job as you. If there's multiple of one job, or a shadow has
+      // a job of 0 (player died), then return '?' for the affected players.
+      id: 'E10S Shadow Of A Hero',
+      netRegex: NetRegexes.addedCombatantFull({ name: 'Shadow Of A Hero' }),
```

Comment:
If we're already using `addedCombatantFull`, is there a particular reason not to use the `npcNameId` field instead? This would remove the need for translation here, and the name of the NPC is in the trigger ID already.

Reply:
How can I find the value for this?

Reply:
It's field 9 in [`addCombatant` 03 log lines](https://github.com/quisquous/cactbot/blob/445188579e4d008f22eb6efb75922758d21fda70/resources/netregexes.js#L160). I typically prefer using this for any `addCombatant` triggers because it doesn't require translations.

Reply:
So, I was about to come back around and do this; however, since we're going to need the name for the below headMarker regex (do we?), should I leave this as is?